oteapi/strategies/resource/application_vnd_postgres.py

```diff
@@ -0,0 +1,115 @@
+"""Strategy class for application/vnd.postgresql"""
+# pylint: disable=unused-argument
+from typing import TYPE_CHECKING, Optional
+
+import psycopg
+from pydantic import Field
+from pydantic.dataclasses import dataclass
+
+from oteapi.models import AttrDict, DataCacheConfig, ResourceConfig, SessionUpdate
+from oteapi.models.resourceconfig import HostlessAnyUrl
+
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Any, Dict
+
+
+class PostgresConnectionConfig(AttrDict):
+    """Configuration data model for
+    [`PostgresResourceStrategy`][oteapi.strategies.resource.application_vnd_postgres.PostgresParseStrategy]."""
+
+    connection_dict: dict = Field(
+        ..., description="Dictionary used for connection with postgres DB via psycopg."
+    )
+    datacache_config: Optional[DataCacheConfig] = Field(
+        None,
+        description="Configuration options for the local data cache.",
+    )
+
+    # TODO seems a bit strange to have the sqlquery here, artifact from
+    #     importing from the sqllite version
+    sqlquery: Optional[str] = Field("", description="A SQL query string.")
+
+
+class PostgresResourceConfig(ResourceConfig):
+    """Postgres resource strategy resource config."""
+
+    # TODO: accessUrl and accessService are required, but in this implmentation do nothing, discuss!
+    accessUrl: HostlessAnyUrl = Field(
```

Comment:
accessUrl and accessService is already defined in ResourceConfig, so not necessary to redefine here. accessUrl will upon instantiation name the host of the service (and optionally all other fields such as port, user, password, database)